e2e/test-subjects/shell-nuxt/app.vue

```diff
@@ -0,0 +1,27 @@
+<template>
+  <div>
+    <h1>Nuxt Host</h1>
+    <NuxtLink to="/">Home page</NuxtLink> | 
+    <NuxtLink to="/vue">Vue page</NuxtLink> | 
+    <NuxtLink to="/custom">Custom page</NuxtLink> |
+    <NuxtLink to="/custom-error">Custom error</NuxtLink>
+    <HostProvider :runtime="runtime" origin="http://localhost:56500">
+      <NuxtPage />
+    </HostProvider>
+  </div>
+</template>
+
+<script setup lang="ts">
+  import { configureRuntime } from '@leanjs/core';
+  import { HostProvider } from "@leanjs/nuxt";
+  const defaultState = {
+    locale: "en",
+  };
+
+  const { createRuntime } =  configureRuntime(defaultState)({
```

Comment:
You can import this directly from https://github.com/leanjs/leanjs/blob/main/e2e/test-subjects/package-runtime-shared/src/index.ts

Reply:
done!